### PR TITLE
Import BDR 3.7.14 docs

### PR DESCRIPTION
**Pull request by jericson-edb**
## What Changed?

* Update BDR 3.7 docs to 3.7.14

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [x] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [x] This PR is for a release, please add this tag: product/bdr/3.7.14

### CODE REVIEW

File: product_docs/docs/bdr/3.7/functions.mdx [hunk 1]
```diff
@@ -57,7 +57,7 @@ connected to.  This allows an application to figure out what node it
 is connected to even behind a transparent proxy.
 
 It is also used in combination with CAMO, see the
-[Connection pools and proxies](camo#connection-pools-and-proxies)
+[CAMO.md#connection-pools-and-proxies]\(Connection pools and proxies)
```
Comment (josh-heyer) [inline product_docs/docs/bdr/3.7/functions.mdx:60]: This link isn't going to work... Something fishy in the source material

File: product_docs/docs/bdr/3.7/repsets.mdx [hunk 2]
```diff
@@ -278,13 +278,11 @@ transaction.
...
     from that replication set.  
-
-    For the same reason, you should not drop a replication set if
+     the same reason, you should not drop a replication set if
     there is a join operation in progress, and the node being joined
     is a member of that replication set; replication set membership is
     only checked at the beginning of the join.  
-
-    This happens because the information on replication set usage is
+    s happens because the information on replication set usage is
```
Comment (josh-heyer) [inline product_docs/docs/bdr/3.7/repsets.mdx:285]: Looks like someone d a word...

File: product_docs/docs/bdr/3.7/striggers.mdx [hunk 3]
```diff
@@ -175,13 +175,13 @@ otherwise data divergence will occur. Technical Support recommends that all conf
...
 
-!!! Warning
-    -   Multiple conflict triggers can be specified on a single table, but
-        they should match distinct event, i.e. each conflict should only
-        match a single conflict trigger.  
-    -   Multiple triggers matching the same event on the same table are
-        not recommended; they might result in inconsistent behaviour, and
-        will be forbidden in a future release.  
+!!!Warning
+-   Multiple conflict triggers can be specified on a single table, but
```
Comment (josh-heyer) [inline product_docs/docs/bdr/3.7/striggers.mdx:179]: Indentation is weird here; verify that this renders.

File: product_docs/docs/bdr/3.7/repsets.mdx (hunk 2 above)
Comment (dougcalobrisi) [reply product_docs/docs/bdr/3.7/repsets.mdx:285]: Oddly, I don't see that in the source material: https://github.com/EnterpriseDB/bdr/blob/548b3d63118861bfbbee0701a144f8457f8978b2/docs/docs2/repsets.md?plain=1#L282

Comment (josh-heyer) [reply product_docs/docs/bdr/3.7/repsets.mdx:285]: Weird; this looks like a bug in the admonitions component: I see now that it did the same thing on line 281 - both are indented with a tab (vs 4 spaces); betting it's assuming there will be a four-space indentation and just eating the first four characters.

**Review by dougcalobrisi — APPROVED** [review]

**Pull request merged**